rpms/group/admin-tools.pan

```diff
@@ -0,0 +1,25 @@
+# Based on SL5.9 group definition
+
+unique template rpms/group/admin-tools;
+
+prefix '/software/packages';
+
+# Default Packages:
+'{authconfig-gtk}' ?= nlist();
```

Comment:
I suggest to use `pkg_repl()` rather than a `nlist` declaration as it allows to detect if an explicit version was declared before. Performance penalty should be very minimal. The syntax is:
```
'/software/packages' = pkg_repl('authconfig-gtk');
```

You should also prefer using `hash()` than `nlist()`: the result is the same but this is the new name adopted a couple of years ago and implemented since panc v10 that is the baseline version now.